include/scrimmage/pubsub/NetworkDevice.h

```diff
@@ -73,19 +73,20 @@ class NetworkDevice {
     }
 
 
-    void set_max_queue_size(unsigned int size);
+    void set_max_queue_size(const unsigned int& size);
```

Comment:
Where applicable (and where it won't break user API), use of const-correctness. I didn't fix the entire scrimmage api, but I making progress as files are modified.

Reply:
Thanks.